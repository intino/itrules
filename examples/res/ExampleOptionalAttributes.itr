def type(Roster)
	<roster>
		$Coach
		<players>
			$Players...[$NL]
		</players>
    </roster>
end

def type(Person) AND trigger(Coach)
    <coach name="$Name" <<year="$Birthday+Year">> country="$Country"<< club="$Club">>/>
end

def type(Person)
    <player name="$Name" year="$Birthday+Year" country="$Country"<< club="$Club">>/>
end
