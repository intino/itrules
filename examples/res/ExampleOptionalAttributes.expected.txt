<roster>
	<coach name="Juan Antonio Orenga" country="Spain"/>
	<players>
		<player name="Pau Gasol" year="" country="Spain" club="L.A. Lakers"/>
		<player name="Rudy Fernandez" year="" country="Spain"/>
		<player name="Juan Carlos Navarro" year="" country="Spain"/>
	</players>
</roster>